src/net/smooth/index.ts

```diff
@@ -175,14 +178,13 @@ export default function(
           segmentInfos: initSegmentInfos,
         });
       }
-      if (__DEV__) {
-        assert(responseData instanceof ArrayBuffer);
-      }
-      const responseBuffer = new Uint8Array(responseData as ArrayBuffer);
+      const responseBuffer = response.responseData instanceof Uint8Array ?
```

Comment:
Why ``response.responseData`` when ``responseData`` is already in your scope and references the same thing?